helm/hpcc/templates/eclagent.yaml

```diff
@@ -16,7 +16,7 @@ data:
     {{ $apptype }}:
 {{ toYaml (omit .me "logging") | indent 6 }}
 {{- include "hpcc.generateLoggingConfig" . | indent 6 }}
-{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" (list "storage" "ecl" "ecl-user" ) ) | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" .secretsCategories ) | indent 6 }}
```

Comment:
I see "secretsCategories passed into dict when hpcc.agentConfigMap called, but comment at top of hpcc.agentConfigMap should be updated.